cpp/src/io/orc/reader_impl.hpp

```diff
@@ -30,15 +27,14 @@
 #include <rmm/cuda_stream_view.hpp>
 
 #include <memory>
-#include <string>
-#include <utility>
+#include <optional>
 #include <vector>
 
 namespace cudf::io::orc::detail {
+using namespace cudf::io::detail;
```

Comment:
I don't know if we should be doing this, especially in a header file.

Reply:
This is due to the changing of namespace that was just merged in, and to minimize code change. Of course I can remove this and add more prefix into the variables.

Reply:
Indeed, there was only one variable that needed to be added prefix. Thanks for rasing this up.

Reply:
thank you for looking into this!